src/tests/suite/daffodil.test.ts

```diff
@@ -142,4 +150,138 @@ suite('Daffodfil', () => {
       assert.strictEqual(daffodilVersion, '0.0.0')
     })
   })
+
+  suite('non-debug specifc commands', () => {
+    const nonDebugSpecificCmds = [
+      'extension.dfdl-debug.debugEditorContents',
+      'extension.dfdl-debug.runEditorContents',
+      'launch.config',
+    ]
+
+    // This breaks when the omega-edit tests run for some reason
```

Comment:
Should a ticket be created for this mystery?

Reply:
Made issue #300 to look into these two unit tests.